backend/src/main/java/asegroup1/api/services/landregistry/LandRegistryServiceImpl.java

```diff
@@ -100,4 +143,20 @@ private JSONObject executeSPARQLQuery(String query) throws UnirestException, IOE
                 .getObject();
     }
 
+
+	private String buildQuery(LandRegistryQuerySelect select, LandRegistryQueryConstraint values) {
+		String query = getQueryPrefixDeclarations() + " " + select.buildQuerySelect() + " " + values.buildQueryWhere();
+
+
+		return query;
+	}
+
+	private String getQueryPrefixDeclarations() {
+		return "prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> " + "prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> "
```

Comment:
Maybe we should store this in the `queries.properties` file?

Reply:
Not sure how the program will benefit, as it will still need to be loaded, making loading take longer, and it would take up more space, due to loading.